src/DataMigration/docs/New-AzDataMigrationToSqlVM.md

```diff
@@ -0,0 +1,563 @@
+---
+external help file:
+Module Name: Az.DataMigration
+online version: https://docs.microsoft.com/powershell/module/az.datamigration/new-azdatamigrationtosqlvm
+schema: 2.0.0
+---
+
+# New-AzDataMigrationToSqlVM
+
+## SYNOPSIS
+Create or Update Database Migration resource.
+
+## SYNTAX
+
+```
+New-AzDataMigrationToSqlVM -ResourceGroupName <String> -SqlVirtualMachineName <String> -TargetDbName <String>
+ [-SubscriptionId <String>] [-AzureBlobAccountKey <String>] [-AzureBlobContainerName <String>]
+ [-AzureBlobStorageAccountResourceId <String>] [-FileSharePassword <String>] [-FileSharePath <String>]
+ [-FileShareUsername <String>] [-Kind <ResourceType>] [-MigrationOperationId <String>]
+ [-MigrationService <String>] [-OfflineConfigurationLastBackupName <String>] [-OfflineConfigurationOffline]
+ [-ProvisioningError <String>] [-Scope <String>] [-SourceDatabaseName <String>]
+ [-SourceSqlConnectionAuthentication <String>] [-SourceSqlConnectionDataSource <String>]
+ [-SourceSqlConnectionEncryptConnection] [-SourceSqlConnectionPassword <String>]
+ [-SourceSqlConnectionTrustServerCertificate] [-SourceSqlConnectionUserName <String>]
+ [-TargetDatabaseCollation <String>] [-TargetLocationAccountKey <String>]
+ [-TargetLocationStorageAccountResourceId <String>] [-DefaultProfile <PSObject>] [-AsJob] [-NoWait]
+ [-PassThru] [-Confirm] [-WhatIf] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Create or Update Database Migration resource.
+
+## EXAMPLES
+
+### Example 1: Start a Database Migration from the on-premise Source Sql Server to target Sql VM
+```powershell
+PS C:\> New-AzDataMigrationToSqlVM -ResourceGroupName "MyResourceGroup" -SqlVirtualMachineName "MyVM" -TargetDbName "MyDb" -Kind "SqlVm" -Scope "/subscriptions/f890983-56793-782e7-378-3563829e/resourceGroups/MyResourceGroup/providers/Microsoft.SqlVirtualMachine/sqlVirtualMachine/MyVM" -MigrationService "/subscriptions/f890983-56793-782e7-378-3563829e/resourceGroups/MyResourceGroup/providers/Microsoft.DataMigration/SqlMigrationServices/MySqlMigrationService" -TargetLocationStorageAccountResourceId "/subscriptions/f890983-56793-782e7-378-3563829e/resourceGroups/MyResourceGroup/providers/Microsoft.Storage/storageAccounts/MyStorageAccount" -TargetLocationAccountKey "dwidjwiwnnwiojnnmskoam==adnkdnwdnwknk" -FileSharePath "\\filesharepath.com\SharedBackup\MyBackUps" -FileShareUsername "filesharepath\User" -FileSharePassword "password" -SourceSqlConnectionAuthentication "SqlAuthentication" -SourceSqlConnectionDataSource "LabServer.database.net" -SourceSqlConnectionUserName "User" -SourceSqlConnectionPassword "password" -SourceDatabaseName "AdventureWorks"
```

Comment:
Please use the fake subscription

Reply:
I will update it to 0000-1111-2222-3333.

Reply:
Updated